manuscript.md

```diff
@@ -14,6 +14,37 @@
 
 ## What is currently enabling our ability to predict network structure?
 
+The acquisition of biodiversity and environmental data has tremendously 
+increased over the past decades thanks to the rise of citizen science 
+[@Dickinson2010CitSci] and of novel technology [@Stephenson2020TecAdv], 
+including wireless sensors [@Porter2005WirSen], DNA monitoring 
+[@Creer2016EcoSF], and satellite remote sensing [@Skidmore2015AgrBio; @Lausch2016LinEar]. 
+Standard practices in data integration and quality control [@Kissling2018BuiEss] and in 
+Next-generation biomonitoring [NGB; @Makiola2020KeyQue] are being set, with favorable 
+consequences on our ability to make reliable predictions of many ecosystem properties 
+and components. Open access databases, such as [GBIF](https://www.gbif.org/) (for biodiversity data), 
+[NCBI](https://www.ncbi.nlm.nih.gov/) (for taxonomic and genomics data), 
+[TreeBASE](https://www.treebase.org/treebase-web/home.html) (for phylogenetics data), 
+[CESTE](https://icestes.github.io/) [@Jeliazkov2020GloDat] (for metacommunity ecology 
+and species traits data), and [WorldClim](https://www.worldclim.org/data/bioclim.html) 
+(for bioclimatic data) contain millions of high-quality data that can be integrated and 
+used to monitor and model biodiversity at the global scale. Regarding species interactions 
+data, [Mangal](https://mangal.io/#/) is perhaps the most comprehensive open database of 
+published ecological networks [@Poisot2016ManMak], whereas 
+[GloBI](https://www.globalbioticinteractions.org/about) is an extensive database 
+of realized and potential species interactions [@Poelen2014GloBio].
+
+The rise of computing power, along with recent advances in machine learning techniques 
+and applications [see @Christin2019AppDee for the use of deep learning in ecology], 
+enable us to manipulate a very large number of data from different sources. Accurate 
+predictions of ecological networks across space can thus be generated by integrating 
+various high-quality, open access datasets, such as the ones archived in the abovementioned 
+databases, if we use predictive methods appropriately. Moreover, ongoing developments in 
+the field of artificial intelligence are aimed at using deep learning more efficiently in 
+low-data regimes [e.g. @Antoniou2018DatAug] and with unbalanced datasets [@Chawla2010DatMin]. 
+Considering the current biases in network ecology [@Poisot2020EnvBia] and the scarcity of data 
```

Comment:
we could maybe add in a bit about coupled with the growth of OA databases the future looks very positive?